app/logical/tag_query.rb

```diff
@@ -1286,8 +1286,8 @@ def parse_query(query, depth: 0, **kwargs)
 
       when "order", "-order"
         q[:order] = g2.downcase
-        q[:order].delete_suffix("_desc") unless q[:order] == "id_desc"
-        q[:order] = ORDER_VALUE_INVERSIONS[q[:order]] if type == :must_not
```

Comment:
This is the 2nd of 2 bugs fixed in commit 351f003; I forgot to convert the `String` to a `Symbol` & not overwrite the assignment if given key wasn't in `TagQuery::ORDER_VALUE_INVERSIONS`. It was fixed by calling `String.to_sym` on the `String` & reassigning the current value if `nil` was returned from `TagQuery::ORDER_VALUE_INVERSIONS`.